src/schemas/execute.yaml

```diff
@@ -0,0 +1,284 @@
+  title: Accounts in state to be overridden
+  type: array
+  items:
+    $ref: '#/components/schemas/AccountOverride'
+AccountOverride:
+  title: Details of an account to be overridden
+  type: object
+  oneOf:
+    - $ref: '#/components/schemas/AccountOverrideState'
+    - $ref: '#/components/schemas/AccountOverrideStateDiff'
+AccountOverrideState:
+  title: Account override with whole storage replacement
+  required:
+    - state
+  properties:
+    address:
+      title: Address
+      $ref: '#/components/schemas/address'
+    nonce:
+      title: Nonce
+      $ref: '#/components/schemas/uint64'
+    balance:
+      title: Balance
+      $ref: '#/components/schemas/uint256'
+    code:
+      title: Code
+      $ref: '#/components/schemas/bytes'
+    moveToAddress:
+      title: MoveToAddress
+      $ref: '#/components/schemas/address'
+    state:
+      title: Storage
+      $ref: '#/components/schemas/AccountStorage'
+AccountOverrideStateDiff:
+  title: Account override with partial storage modification
+  required:
+    - stateDiff
+  properties:
+    address:
+      title: Address
+      $ref: '#/components/schemas/address'
+    nonce:
+      title: Nonce
+      $ref: '#/components/schemas/uint64'
+    balance:
+      title: Balance
+      $ref: '#/components/schemas/uint256'
+    code:
+      title: Code
+      $ref: '#/components/schemas/bytes'
+    moveToAddress:
+      title: MoveToAddress
+      $ref: '#/components/schemas/address'
+    stateDiff:
+      title: Storage difference
+      $ref: '#/components/schemas/AccountStorage'
+AccountStorage:
+  title: Storage slots for an account
+  type: object
+  additionalProperties:
+    - $ref: '#/components/schemas/hash32'
+BlockOverrides:
+  title: Context fields related to the block being executed
+  type: object
+  properties:
+    number:
+      title: Number
+      $ref: '#/components/schemas/uint256'
+    prevRandao:
+      title: Randomness beacon
+      $ref: '#/components/schemas/uint256'
+    time:
+      title: Time
+      $ref: '#/components/schemas/uint256'
+    gasLimit:
+      title: Gas limit
+      $ref: '#/components/schemas/uint64'
+    feeRecipient:
+      title: feeRecipient
+      $ref: '#/components/schemas/address'
+    baseFeePerGas:
+      title: Base fee per gas
+      $ref: '#/components/schemas/uint256'
+MultiCallResult:
+  title: Full results of multi call
+  type: array
+  items:
+    $ref: '#/components/schemas/MultiCallBlockResult'
+MultiCallBlockResult:
+  title: Result of multicall block-level, with array of calls
+  type: object
+  properties:
+    number:
+      title: Number
+      $ref: '#/components/schemas/uint256'
+    hash:
+      title: block hash
+      $ref: '#/components/schemas/hash32'
+    timestamp:
+      title: Time
+      $ref: '#/components/schemas/uint256'
+    gasLimit:
+      title: Gas limit
+      $ref: '#/components/schemas/uint64'
+    gasUsed:
+      title: Gas used
+      $ref: '#/components/schemas/uint64'
+    feeRecipient:
```

Comment:
Nitpick: I realize the engine API and CL folks call this `feeRecipient` but on the EL side we have `eth_coinbase` or `eth_getBlockBy*` returns a `miner` field. Now `miner` is outdated but I have a slight preference towards `coinbase` instead of introducing a 3rd name for it.

Reply:
I agree with this, it seemed like feeRecipient was going to be the one that everyone ran with right after 1559, but it slowly made its way back to coinbase. Any disagreement @MicahZoltu ?

Reply:
I pretty strongly favor accurate names rather than names that match internal implementation details that exist for legacy reasons.  In this case, `coinbase` dates back to early Bitcoin days and I *think* it was when your client held your private keys (not sure on that last part).  `feeRecipient` on the other hand is a term that most accurately reflects what the field means.

That being said, if `eth_getBlockBy*` returns the `feeRecpient` address in a field called `coinbase`, I can see the value in being consistent across calls.  It just strikes me as incredibly unfortunate that we seem to be stuck using terrible names forever, making it harder for every new developer in Ethereum to figure out how things work.